### ISSUE TITLE
option/field based price critical issue

### ISSUE THREAD

**Issue by jojouser** [microweber/microweber#275]
Now MW support add custom fields for products. but it have a critical bug(or lack of key function).
for example:
i add a product iphone, and set the price is $599, then i add a radio button(4.7inch and 5.5inch), then i cann't change the price based on the radio. (if clients choose 4.7inch, then the price should dynamic change to $599 and if clients choose 5.5inch, then the price should dynamic change to $699 )

**Comment by nickwuk** [comment]
I agree this is an important feature to be able to optionally set in admin an incremental price for any attribute (in osCommerce the attribute price can be positive or negative) and for the base price to dynamically change according to attribute selection. It would also be useful to be able to set a product sub-code per attribute which is appended to the base code.

**Comment by ash-rain** [comment]
Hi, it's surely possible to do this. However, I'm unable to get into your specific issue right now. Please wait until next week and we'll get back to you in more detail.
Until then keep in mind you can have several prices for the same product. I'm sorry to ask you but please try to figure it out for yourself if time is a factor (and keep in mind there is still a bit of work to be done until stable release).

**Comment by jojouser** [comment]
I have two small website using other cms many years ago. i want to migrate to MW. Now i'm using MW local to do a review. It seams a great tool.

**Comment by ash-rain** [comment]
Sounds great. We're working on some very cool e-commerce features so stay tuned for v.1.0

**Comment by peter-mw** [comment]
Hi, this is important to have. Will try to integrate it before v.1

**Comment by jojouser** [comment]
Any news about this issue?thanks.

**Comment by jojouser** [comment]
Hi, Any news about this issue? It seams the new version 1.0.5 still have this issue. thanks.

**Comment by peter-mw** [comment]
hi @jojouser i haven't started the work on it yet

**Comment by angelorc** [comment]
Any news on this feature?

**Comment by peter-mw** [comment]
We will try to push this for next version 1.0.9

**Comment by angelorc** [comment]
Explain me how you want to integrate it and I'll try to do it

**Comment by peter-mw** [comment]
Ok for example i will make one new column in the table `custom_fields_values` called `value_modifier`

https://github.com/microweber/microweber/blob/1.0.8/src/Microweber/Install/Schema/Content.php#L155


Then in `update_cart` function https://github.com/microweber/microweber/blob/1.0.8/src/Microweber/Providers/Shop/CartManager.php#L309 


i will check for this `value_modifier` here https://github.com/microweber/microweber/blob/1.0.8/src/Microweber/Providers/Shop/CartManager.php#L430-L433


something like

```
 if ($cf['value'] != '') {
if (isset($cf['value_modifier'])) {
 $cf['value'] =  $cf['value'] + floatval( $cf['value_modifier'])
}
$prices[ $cf['name'] ] = $cf['value'];
 }
```

something like that...

**Comment by ozanmora** [comment]
Hello, 

Can you have a solution this issue?
I need this feature on my project. (I'm using Microweber 1.1.2.1)

Thanks & Best Regards

**Comment by skystian** [comment]
Any news on this feature?
